contrib/network/connection_balance/dlb/test/config_test.cc

```diff
@@ -41,17 +41,20 @@ TEST_F(DlbConnectionBalanceFactoryTest, MakeDefaultConfig) {
   makeDlbConnectionBalanceConfig(typed_config, dlb);
   verifyDlbConnectionBalanceConfigAndUnpack(typed_config, dlb);
   EXPECT_EQ(0, dlb.id());
+  EXPECT_EQ(0, dlb.retry_times());
 }
 
 TEST_F(DlbConnectionBalanceFactoryTest, MakeCustomConfig) {
   envoy::config::core::v3::TypedExtensionConfig typed_config;
 
   envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb dlb;
   dlb.set_id(10);
+  dlb.set_retry_times(12);
 
   makeDlbConnectionBalanceConfig(typed_config, dlb);
   verifyDlbConnectionBalanceConfigAndUnpack(typed_config, dlb);
   EXPECT_EQ(10, dlb.id());
+  EXPECT_EQ(12, dlb.retry_times());
```

Comment:
Just a suggestion: these tests do not actually test that the feature performs as it should, they just test that the proto works. You may want to consider testing the actual code.

Reply:
Thanks for your suggestion, I will do it when mock functions are ready.